src/integration/lambda.ts

```diff
@@ -57,9 +57,11 @@ export class LambdaIntegration extends Integration {
     this.fn = fn;
   }
 
-  /** Allow Lambda invoke action to be performed by given identity. */
-  public grantFunctionInvoke(identity: IGrantable): void {
-    this.fn.grantInvoke(identity);
+  public grantFunctionInvoke(scope: Construct, id: string, principal: ServicePrincipal): void {
```

Comment:
It might be useful to document this method in the context of authorizer functions vs. api gateway access